### PR TITLE
Releases/v0.12.latest

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_stripe_source/issues/80 by way of including https://github.com/fivetran/dbt_stripe_source/pull/81

**This PR will result in the following new package version:**

v0.12.1

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Feature Updates
- Declaration of passthrough variables for the `stg_stripe__card` model. This can, for example, be used to pull in non-standard columns from Stripe such as `description`, `iin` and `issuer`. See the [README](https://github.com/fivetran/dbt_stripe_source?tab=readme-ov-file#passing-through-additional-fields) for more details ([PR #81](https://github.com/fivetran/dbt_stripe_source/pull/81)).
```yml
# dbt_project.yml

vars:
  stripe_source:
    card_pass_through_columns:
      - name: "description"
```

## Under the Hood
- Removed un-used columns from the `get_card_columns()` [macro](https://github.com/fivetran/dbt_stripe_source/blob/main/macros/get_card_columns.sql) and aliased the `metadata` field, which is parsed but not wholly included in `stg_stripe__card` by default.
  - This ensures users can utilize the new `card_pass_through_columns` variable to include these fields in `stg_stripe__card`.
 
### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Ran successfully with the following set of passthrough columns:

```yml
  stripe_source:
      card_pass_through_columns:
        - name: "description"
        - name: iin
          transform_sql: cast(iin as string)
        - name: exp_year
          alias: exp_year_alias
          transform_sql: exp_year_alias * 2.0
        - name: metadata
        - name: fingerprint

      stripe__card_metadata:
        - name: external_id
 ```

### If you had to summarize this PR in an emoji, which would it be?
 
🍏

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+# dbt_project.yml
+
+vars:
+  stripe_source:
+    card_pass_through_columns:
+      - name: "description"
+```
+
+## Under the Hood
+- Removed un-used columns from the `get_card_columns()` [macro](https://github.com/fivetran/dbt_stripe_source/blob/main/macros/get_card_columns.sql) and aliased the `metadata` field, which is parsed but not wholly included in `stg_stripe__card` by default ([PR #83](https://github.com/fivetran/dbt_stripe_source/pull/83)).
```
Comment (fivetran-reneeli) [inline CHANGELOG.md:15]: Would you mind clarifying the metadata aliasing portion? Or what does 'not wholly included' mean

**Review by fivetran-reneeli — APPROVED** [review]
Looks good just had one clarifying question in the changelog!

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-jamie) [reply CHANGELOG.md]: How's this?

> - Removed un-used columns from the `get_card_columns()` [macro](https://github.com/fivetran/dbt_stripe_source/blob/main/macros/get_card_columns.sql) and aliased the `metadata` field, which is parsed via the `stripe__card_metadata` [variable](https://github.com/fivetran/dbt_stripe_source?tab=readme-ov-file#pivoting-out-metadata-properties), but not included as a field in its entirety in `stg_stripe__card` by default ([PR #83](https://github.com/fivetran/dbt_stripe_source/pull/83)).

Comment (fivetran-reneeli) [reply CHANGELOG.md]: that reads well! 👍

File: integration_tests/dbt_project.yml [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+version: '0.12.1'
 
 
 profile: 'integration_tests'
 
 vars:
   stripe_schema: stripe_source_integrations_tests_6
+  
   stripe_source:
+    card_pass_through_columns:
```
Comment (fivetran-avinash) [inline integration_tests/dbt_project.yml:13]: @fivetran-jamie Should we remove this? I assume it was only for testing.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+```yml
+# dbt_project.yml
+
+vars:
+  stripe_source:
+    card_pass_through_columns:
+      - name: "description"
+```
+
+## Under the Hood
```
Comment (fivetran-avinash) [inline CHANGELOG.md:19]: @fivetran-jamie I think something is up with the triple quoting here that is causing the Under the Hood section to be quoted.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,23 @@
+# dbt_stripe_source v0.12.1
+
+## Feature Updates
+- Declaration of passthrough variables for the `stg_stripe__card` model. This can, for example, be used to pull in non-standard columns from Stripe such as `description`, `iin` and `issuer`. See the [README](https://github.com/fivetran/dbt_stripe_source?tab=readme-ov-file#passing-through-additional-fields) for more details ([PR #81](https://github.com/fivetran/dbt_stripe_source/pull/81)).
+```yml
+# dbt_project.yml
```
Comment (fivetran-avinash) [inline CHANGELOG.md:7]: should we mirror the configuration that's in the README or is that too much information to translate here?

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-jamie A few source notes before approval.

File: CHANGELOG.md (hunk 4 above)
Comment (fivetran-avinash) [reply CHANGELOG.md:7]: Looks fine in Github, so what do I know.

File: integration_tests/dbt_project.yml (hunk 2 above)
Comment (fivetran-jamie) [reply integration_tests/dbt_project.yml:13]: Figured there's no harm in including them in integration tests moving forward 🤷

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-jamie lgtm

**Pull request merged**